cfme/infrastructure/provider/__init__.py

```diff
@@ -191,14 +186,12 @@ def hosts(self):
 
     def get_clusters(self):
         """returns the list of clusters belonging to the provider"""
-        web_clusters = []
+        collection = ClusterCollection()
```

Comment:
provide appliance arg to ClusterCollection

Reply:
there is @psav PR which fixes ClusterCollection and etc.

Reply:
psavs Pr was merged. So, this is not relevant any longer

Reply:
:+1: everywhere! :)